workspace-manager-clienttests/src/main/resources/suites/BasicNightlyPerf.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "BasicNightlyPerf",
+  "description": "Nightly perf tests.",
```

Comment:
If the schedule is independent of this config, then I'd simply name it `Perf Tests`.

Reply:
Renamed the config and updated README to remove the suggestion of schedule dependency.